config.json

```diff
@@ -572,7 +572,13 @@
       "core": false,
       "difficulty": 6,
       "slug": "bowling",
-      "topics": null,
+      "topics":  [
+        "functions",
+        "mathematics",
+        "games",
+        "conditionals",
+        "integers"
```

Comment:
Great suggestions! :) Maybe also `exception_handling` since the tests expect exceptions to be thrown?

Reply:
oh yeah